src/styles/blocks/contact-us.scss

```diff
@@ -0,0 +1,115 @@
+.contact-us {
+  margin-bottom: 64px;
+
+  &__grid {
+    @include pageGrid;
+  }
+
+  &__form {
+    margin-bottom: 48px;
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 6;
+      margin-bottom: 80px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 0;
+    }
+  }
+
+  &__contacts {
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 6;
+      margin-bottom: 80px;
+    }
+
+    @include onDesktop {
+      grid-column: 8 / 13;
+      margin-bottom: 0;
+    }
+  }
+
+  &__input {
+    width: 100%;
+    height: 48px;
+    box-sizing: border-box;
+    padding: 14px 16px;
+
+    background: #f9f9f9;
+    border: 1px solid transparent;
+    border-radius: 8px;
+
+    font-size: 14px;
+    line-height: 140%;
+    letter-spacing: 0.15px;
+    color: #1b2129;
+
+    transition: all 0.3s;
+
+    &:not(:last-child) {
+      margin-bottom: 16px;
+    }
+
+    &::placeholder {
+      font-family: "'Manrope'", sans-serif;
+      color: #7e7e83;
+      transition: all 0.3s;
+    }
+
+    &:hover {
+      border: 1px solid #d5d7de;
```

Comment:
Consider using variables for colors like #d5d7de for consistency and maintainability.